Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -204,14 +206,13 @@ public void AdjustHappiness( float deltaHappiness )
 		{
 			_happiness = 0;
 		}
-
-		RecalculateStat();
-
-		if ( _happiness < 0 )
+		else if ( _happiness > 1 )
 		{
-			_happiness = 0;
+			_happiness = 1;
```

Comment:
I tested on my own and got an average of 6ms faster with f, but maybe my test was flawed...Or that guy could be on a newer version of .NET. It doesn't matter really